app/src/main/java/no/usn/mob3000/ui/components/DangerousActionDialogue.kt

```diff
@@ -66,15 +64,21 @@ fun DangerousActionDialogue(
                 ) {
                     Button(
                         onClick = onDismiss,
-                        colors = ButtonDefaults.buttonColors(colorButtonDismiss),
+                        colors = ButtonDefaults.buttonColors(
+                            containerColor = MaterialTheme.colorScheme.primary,
+                            contentColor = MaterialTheme.colorScheme.onPrimary
+                        ),
                         modifier = Modifier
                             .weight(1f)
                             .padding(end = 8.dp)
                     ) { Text(dismissText) }
 
                     Button(
                         onClick = onConfirm,
-                        colors = ButtonDefaults.buttonColors(colorButtonConfirm),
+                        colors = ButtonDefaults.buttonColors(
+                            containerColor = Color(0xFFFF0000),
+                            contentColor = Color.White
+                        ),
```

Comment:
This is now less flexible than before. Not only is it hard-coded, but it's also using hexadecimal color codes when a simple `Color.Red` would've sufficed as before. But, again, refer to [this comment](https://github.com/frigvid/mob3000-group11/pull/19/files#r1840946578).